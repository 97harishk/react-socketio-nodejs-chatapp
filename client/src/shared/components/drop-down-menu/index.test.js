import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	DropDownMenuComponent
} from 'shared/components';

describe('<DropDownMenuComponent />', () => {
	const modelI = {};

	const getWrapper = (model) => {
		return shallow((
			<DropDownMenuComponent {...model} />
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
