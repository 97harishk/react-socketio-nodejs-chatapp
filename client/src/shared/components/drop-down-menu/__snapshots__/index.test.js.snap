// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DropDownMenuComponent /> renders correctly 1`] = `
<div>
  teste
</div>
`;
